### ISSUE TITLE
[BUG] - Cannot switch off TraceMempool with 1.34.0

### ISSUE THREAD

**Issue by seriru** [input-output-hk/cardano-node#3682]
**Internal/External**
*External*  

**Area**
*Other* Cannot switch off tracemempool

**Summary**
It's not possible to switch off TraceMempool

**Steps to reproduce**
1. Start the node with TraceMempool set to false (or true; it doesn't matter in my case)
2. curl -s localhost:<prometheus_port>/metrics | grep mem
    returns an entry for cardano_node_metrics_mempoolBytes_int which shouldn't be there.

**Expected behavior**
Shouldn't see cardano_node_metrics_mempoolBytes_int entry in 2.

**System info (please complete the following information):**
- OS Name: [CentOS Stream 8]
- OS Version [EL8]
- Node version (output of `cardano-node --version`)
cardano-node 1.34.0 - linux-x86_64 - ghc-8.10
git rev c23d5d319cd3276575c6ac32458516232e8d2c48
- CLI version  (output of `cardano-cli --version`)
n/a

**Screenshots and attachments**
mainnet-config.json
```
{
  "AlonzoGenesisFile": "mainnet-alonzo-genesis.json",
  "AlonzoGenesisHash": "7e94a15f55d1e82d10f09203fa1d40f8eede58fd8066542cf6566008068ed874",
  "ApplicationName": "cardano-sl",
  "ApplicationVersion": 1,
  "ByronGenesisFile": "mainnet-byron-genesis.json",
  "ByronGenesisHash": "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb",
  "LastKnownBlockVersion-Alt": 0,
  "LastKnownBlockVersion-Major": 3,
  "LastKnownBlockVersion-Minor": 0,
  "MaxKnownMajorProtocolVersion": 2,
  "Protocol": "Cardano",
  "RequiresNetworkMagic": "RequiresNoMagic",
  "ShelleyGenesisFile": "mainnet-shelley-genesis.json",
  "ShelleyGenesisHash": "1a3be38bcbb7911969283716ad7aa550250226b76a61fc51cc9a9a35d9276d81",
  "TraceAcceptPolicy": true,
  "TraceBlockFetchClient": false,
  "TraceBlockFetchDecisions": true,
  "TraceBlockFetchProtocol": false,
  "TraceBlockFetchProtocolSerialised": false,
  "TraceBlockFetchServer": false,
  "TraceChainDb": true,
  "TraceChainSyncBlockServer": false,
  "TraceChainSyncClient": false,
  "TraceChainSyncHeaderServer": false,
  "TraceChainSyncProtocol": false,
  "TraceConnectionManager": true,
  "TraceDNSResolver": true,
  "TraceDNSSubscription": true,
  "TraceDiffusionInitialization": true,
  "TraceErrorPolicy": true,
  "TraceForge": true,
  "TraceHandshake": false,
  "TraceInboundGovernor": true,
  "TraceIpSubscription": true,
  "TraceLedgerPeers": true,
  "TraceLocalChainSyncProtocol": false,
  "TraceLocalErrorPolicy": true,
  "TraceLocalHandshake": false,
  "TraceLocalRootPeers": true,
  "TraceLocalTxSubmissionProtocol": false,
  "TraceLocalTxSubmissionServer": false,
  "TraceMempool": false,
  "TraceMux": false,
  "TracePeerSelection": true,
  "TracePeerSelectionActions": true,
  "TracePublicRootPeers": true,
  "TraceServer": true,
  "TraceTxInbound": false,
  "TraceTxOutbound": false,
  "TraceTxSubmissionProtocol": false,
  "TracingVerbosity": "MinimalVerbosity",
  "TurnOnLogMetrics": true,
  "TurnOnLogging": true,
  "defaultBackends": [
    "KatipBK"
  ],
  "defaultScribes": [
    [
      "FileSK",
      "/data/cardano/logs/relay.log"
    ]
  ],
  "hasEKG": 12601,
  "hasPrometheus": [
    "0.0.0.0",
    12701
  ],
  "minSeverity": "Info",
  "traceForwardTo": {
     "tag": "RemoteSocket",
     "contents": [
        "localhost",
        "4002"
     ]
  },
  "options": {
    "mapBackends": {
      "cardano.node.metrics": [
          "EKGViewBK",
	  "TraceForwarderBK"
      ],
      "cardano.node.resources": [
          "EKGViewBK",
	  "TraceForwarderBK"	  
      ]
    },
    "mapSubtrace": {
      "cardano.node.metrics": {
        "subtrace": "Neutral"
      }
    }
  },
  "rotation": {
    "rpKeepFilesNum": 10,
    "rpLogLimitBytes": 5000000,
    "rpMaxAgeHours": 24
  },
  "setupBackends": [
      "KatipBK",
      "TraceForwarderBK"
  ],
  "setupScribes": [
    {
      "scFormat": "ScJson",
      "scKind": "FileSK",
      "scName": "/data/cardano/logs/relay.log",
      "scRotation": null
    }
  ]
}

```

**Comment by btbf** [comment]
#3668 
Already fixed.
1.34.1 should be released shortly.

**Comment by seriru** [comment]
I tested my configuration on 1.34.1. Still unable to disable TraceMempool.

**Comment by btbf** [comment]
```cardano-node version```
cardano-node 1.34.1 - linux-x86_64 - ghc-8.10
git rev 73f9a746362695dc2cb63ba757fbcabb81733d23

```curl -s localhost:<prometheus_port>/metrics | grep mem```
There is also no return value for this command

I was able to disable it in my environment

**Comment by seriru** [comment]
My mistake. It's fixed in 1.34.1.
